src/include/robin_hood.h

```diff
@@ -367,8 +438,16 @@ class BulkPoolAllocator {
     }
 
     // enforce byte alignment of the T's
-    static constexpr size_t ALIGNMENT =
+#if __cplusplus >= 201402L
+    static const size_t ALIGNMENT =
         (std::max)(std::alignment_of<T>::value, std::alignment_of<T*>::value);
+#else
+    static const size_t ALIGNMENT =
+        (ROBIN_HOOD_STD_COMP::alignment_of<T>::value > ROBIN_HOOD_STD_COMP::alignment_of<T*>::value)
+            ? ROBIN_HOOD_STD_COMP::alignment_of<T>::value
+            : +ROBIN_HOOD_STD_COMP::alignment_of<T*>::value; // the + is for walkarround
```

Comment:
I don't understand the `+` and what do you mean by walkaround?

Reply:
I will cause a duplicate branch warning, blahblah, and then with "treat warning as error" enabled, caused a problem with my MSVC build.